src/main/java/serhat/workshop_3/Task4.java

```diff
@@ -0,0 +1,43 @@
+package serhat.workshop_3;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Created by serhatgemici on 13/02/2017.
+ */
+public class Task4 {
+
+    public static void main(String[] args) {
+        extractDigitsFromText("I'm 40-year-old. I have 5 cats and one fish.");
+        extractDigitsFromText("Lorem Ipsum");
+        extractDigitsFromText("this is 34.6");
+    }
+
+    private static void extractDigitsFromText(String text) {
+        List wordList = new ArrayList();
+        Pattern pattern = Pattern.compile("\\d+(\\.\\d+)?");
+        Matcher matcher = pattern.matcher(text);
+
+        while (matcher.find()) {
+
+            wordList.add(matcher.group());
+
+        }
+
+        if (wordList.isEmpty()) {
```

Comment:
Try to avoid usage of multiple System.out.println()
Better to concatenate string using formatting, like:

if (wordList.isEmpty()) {
            printResult(text, "No digits were found");
        } else {
            printResult(text, "Digits found: " + wordList);
}

...

private static void printResult(String text, String result) {
        String pattern = "Text:\n%s\n%s\n";
        System.out.println(String.format(pattern, text, result));
}